docs/source/usage.rst

```diff
@@ -185,6 +186,16 @@ or
   id3 id1 id2
   id4 id1 id2
 
+If work on sex chromosome. 0 refer to male, and 1 refer to female.
```

Comment:
When working with X chromosome use this format - add sex information in the fourth column (0 indicates male and 1 indicates female):